### DISCUSSION TITLE
Show as attributes the aggregated sensors

### DISCUSSION THREAD

**Discussion by mihsu81** [jseidl/magic-areas#364] [Ideas / Feature Requests]
@jseidl Would it be possible to add aggregated sensors as attributes of the main sensor, in a similar way as the Area State entity?
It would help identify which sensors of a certain device class should be excluded.
I.e. identify device temperature sensors added to the aggregated sensor.
I might be wrong but I think this was present in a previous major revision of MA.

**Comment by jseidl** [comment]
Yes there used to be an attribute called `sensors` but that was duplicated so I've removed. If you look at the entity under the "states" section on "developer tools" you will see an attribute called entity ID. 

![Screenshot_2024-06-10-08-49-06-81_c3a231c25ed346e59462e84656a70e50](https://github.com/jseidl/hass-magic_areas/assets/283041/369731ca-7719-4f6c-8db0-65d57b6bf999)

**Reply by mihsu81** [reply]
Wasn't aware that they would only appear in developer tools. Would it be possible to make them appear also when viewing the sensor details?
Thanks in advance.
![image](https://github.com/jseidl/hass-magic_areas/assets/40715497/3a3857ab-cd50-4a2f-8856-e653af9388ae)
![image](https://github.com/jseidl/hass-magic_areas/assets/40715497/a0bab1a5-e4d7-4bde-a00f-800b5bb8ea1c)

**Reply by jseidl** [reply]
> Would it be possible to make them appear also when viewing the sensor details?

I need to investigate. So far the only way would be duplicating the key under another name which I want to avoid.

**Reply by TimSoethout** [reply]
Came here looking for this. Great to see it is available in the developer tools.
I thought attributes by default were visible in the sensor details. Why doesn't `entity_id` show up then?
